test/rollupBridge.js

```diff
@@ -0,0 +1,550 @@
+
+/**
+ * Copyright (c) 2018-present, Leap DAO (leapdao.org)
+ *
+ * This source code is licensed under the Mozilla Public License, version 2,
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+import { Tx, Input, Output, Outpoint, Block } from 'leap-core';
+
+require('./helpers/setup');
+
+const Bridge = artifacts.require('RollupBridge');
```

Comment:
Next time call it RollupBridge hehe. I know this is not getting merged, but it is a good opportunity to write something and prove I actually reviewed this. 💯